app/twisty/cube3d-stickering.ts

```diff
@@ -38,6 +38,7 @@ addAlg("VLS", "R' F2 R F2' L' U2 L");
 addAlg("LS", "U' R U' R' U R U R'");
 addAlg("EO", "R' F R");
 addAlg("CMLL", "F R U R' U' F'");
+addAlg("COLL", "L R' U' R U L' U2 R' U2 R");
```

Comment:
How would you feel about moving this between OLL and ELS like in the other files?

Reply:
I'm about to do a release, so I'm going to go ahead and merge and move this myself. Hope that's okay!